src/client/storage/default_cache.rs

```diff
@@ -0,0 +1,68 @@
+
+use {
+    super::{RemoteStorageError, RemoteStorage},
+    log::{debug, info},
+    serde_json::Value,
+    std::path::Path,
+    std::io::prelude::*,
+    std::fs::OpenOptions,
+};
+
+pub struct DefaultCache {}
+
+impl RemoteStorage for DefaultCache {
+    fn store(&self, key: &str, value: Value) -> Result<(), RemoteStorageError> {
+        debug!("default cache store key={},value={}", key, value);
+
+        let file_name = &format!("{}.txt", key);
+        let path = Path::new(file_name);
+        let display = path.display();
+
+        // Open the path in read-only mode, returns `io::Result<File>`
```

Comment:
`.write(true)` does not sound like read-only ;)

Reply:
lol, I forgot to remove the comment; its supposed to have write permissions :)